docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

```diff
@@ -0,0 +1,40 @@
+[id="proc_adding-a-volume-component-to-a-devfile_{context}"]
+= Adding a volume component to a devfile
+
+[role="_abstract"]
+You can use a volume component to share files among container components and collaborate with other teams during the development process.
+
+.Prerequisites
+
+* xref:adding-schema-version-to-a-devfile.adoc[]
+* xref:adding-a-name-to-a-devfile.adoc[]
+* xref:adding-components-to-a-devfile.adoc[]
+
+.Procedure
+
+* Store an ephemeral `volume` component for it to be persistent across restarts.
```

Comment:
This is not a procedure.

Reply:
Thanks, Robert. I'm not sure why this isn't a procedure. Like, how `adding a volume component` isn't a procedure but `adding a container component` and `adding a kubernetes or openshift component` are procedures? Why don't you think it is? 

I do see that the title is `adding a volume component` but my only step says `store an ephemeral volume component`. So I changed it to `Add an ephemeral volume component...` to be consistent. 

Thanks for your time and review!

Reply:
I didn't mean that the whole thing wouldn't be a procedure -- the comment was about the instruction only. Your change makes it easier to understand. Sorry for not providing enough info/context.

On a related note: is the component really 'ephemeral'? It seems to me that the act of adding it to the devfile makes it persistent -- or is that the intended meaning?

Reply:
Thanks, please see the new commit and let me know what you think!